google/cloud/forseti/services/cli.py

```diff
@@ -1009,11 +1028,30 @@ def __init__(self, *args, **kwargs):
         """
         super(DefaultConfig, self).__init__(*args, **kwargs)
 
+        self.DEFAULT['endpoint'] = self.get_default_endpoint()
+
         # Initialize default values
         for key, value in self.DEFAULT.iteritems():
             if key not in self:
                 self[key] = value
 
+    def get_default_endpoint(self):
+        """Get server address from the forseti_client_conf.yaml file
```

Comment:
To be accurate, this description should be shorter ```Get server address``` as it could either be the conf file OR the default value```